### PR TITLE
Improve report so that the message will be shown

### PR DESCRIPTION
**Pull request by panpawn**
Before the message instructing the user to join ask in the help room was not readable since the command was making you join Help immediately.  I did this by changing the way the message gets sent (now as a popup after joining the room). Additionally, I have also slightly changed the message to be more helpful to the user.

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -95,10 +95,10 @@ var commands = exports.commands = {
 	requesthelp: 'report',
 	report: function (target, room, user) {
 		if (room.id === 'help') {
-			this.sendReply("Ask one of the Mods in the Help room.");
+			this.sendReply("Ask one of the Moderators (@) in the Help room.");
 		} else {
-			this.sendReply("Ask a Mod in the Help room.");
 			this.parse('/join help');
+			user.popup("Ask one of the Moderators (@) in the Help room.");
```
Comment (Rhythms88) [inline commands.js]: this shouldn't be a popup. In fact, it shouldn't really be there at all.

**Comment by panpawn** [comment]
It had a message before, I was just improving it so that it was actually useful.

**Comment by Rhythms88** [comment]
I know it had a message... I added it.

File: commands.js (hunk 1 above)
Comment (Zarel) [inline commands.js]: ^

**Comment by ascriptmaster** [comment]
Basically, as mentioned at https://github.com/Zarel/Pokemon-Showdown/pull/1853#discussion_r29559196 that message should be removed entirely. No popup or anything.

The other message is a good change, though.

File: commands.js (hunk 1 above)
Comment (xfix) [inline commands.js]: But if a message would be closed, and user is already in Help, nothing would happen, and this could actually confuse an user.

**Comment by Rhythms88** [comment]
Seeing as @Zarel did this change in https://github.com/Zarel/Pokemon-Showdown/commit/b322cccc59e82381f96b19412747167c9fb0bf36, this PR is no longer necessary

**Pull request closed without merging**